dev/CommandBarFlyout/CommandBarFlyoutCommandBar.cpp

```diff
@@ -471,16 +475,19 @@ void CommandBarFlyoutCommandBar::UpdateFlowsFromAndFlowsTo()
 }
 
 void CommandBarFlyoutCommandBar::UpdateUI(
-    bool useTransitions)
+    bool useTransitions, bool isForCommandBarElementDependencyPropertyChange)
 {
+    COMMANDBARFLYOUT_TRACE_VERBOSE(*this, TRACE_MSG_METH_INT_INT, METH_NAME, this, useTransitions, isForCommandBarElementDependencyPropertyChange);
```

Comment:
>COMMANDBARFLYOUT_TRACE_VERBOSE [](http://example.com/codeflow?start=4&length=30)

should this be commented out as well?

Reply:
This one is fine, without literals. I added "#ifdef _DEBUG" like ScrollView::OnPropertyChanged and others do.